README.md

```diff
@@ -1,4 +1,23 @@
-# DeepPocketsHomework
-This is a place to put any files or non-code work you are doing for the DeepPockets project.
+# Here is my database schema.
 
-Add Your own work to your own branch and if you want to have others review it, create a PR for the REVIEWS branch (these will never be merged).
+You can find the diagram [here](https://dbdiagram.io/d/DP-660b4a1a37b7e33fd740cea8)
+
+![schema](schema.png)
+
+# Explanation
+
+Each budget has at least one dedicated account to store money. 
+There are different types of budget (regular, goal, etc.)
+
+Account keeps monet and can belong to only budget.
```

Comment:
Thanks for you review!
I think my explanation was not clear enough.

```In transaction, from_account_id and to_account_id should be nullable.``` that's right. They are nullable actually:

![image](https://github.com/DevMentorDave/DeepPocketsHomework/assets/65569159/1f223903-abc7-417a-aa55-d633148b1326)

![image](https://github.com/DevMentorDave/DeepPocketsHomework/assets/65569159/19c46d25-8121-41ea-aee6-cee79846e507)

This diagram editor allows to set `not null`, so all fields are nullable by default.

```I don't think a link between transaction and scheduled_transation is needed.```
I think that it is needed because the app should be able to distinguish scheduled transaction for the purpose of checking if it was done . If I have a planned transaction and the app is going to do it, first the app should be able to check if it was done already. 

```A scheduled_transaction can between two different accounts or not. Should it be linked to a budget? ``` It is a good point. Now I think, that scheduled transaction should be linked to account only not to budget. An account is linked to particular budget already. Otherwise it is going to mess different layers of abstraction. In my mind money and transactions are the base level of abstraction. A budget is a hire layer on top of money. 
There should be a rule that a scheduled transaction belongs to an account money go from, if it is earning and there is null in `from_account_id` than a scheduled transaction belongs to an account money go to.

```I don't think we need a schedule_type. This type is only in the application logic.``` Somehow the app should know which logic to apply to a particular scheduled transaction. So I think schedule_type table  has a finite  set of types (weekly, biweekly, monthly, daily, one-time). For each of that types there should be specific functionality in the app. Maybe for that table id, name and description is enough. 

p.s.: I am not able to push any commit
```
remote: error: GH013: Repository rule violations found for refs/heads/db-shema-cootok.
remote: Review all repository rules at http://github.com/DevMentorDave/DeepPocketsHomework/rules?ref=refs%2Fheads%2Fdb-shema-cootok
remote:
remote: - Changes must be made through a pull request.
remote:
To https://github.com/DevMentorDave/DeepPocketsHomework.git
 ! [remote rejected] db-shema-cootok -> db-shema-cootok (push declined due to repository rule violations)
error: failed to push some refs to 'https://github.com/DevMentorDave/DeepPocketsHomework.git'
```